app/ProductPage/page.js

```diff
@@ -0,0 +1,70 @@
+"use client";
+import Image from "next/image";
+import { db } from "@/FirebaseConfig/Firebase";
+import { collection, getDocs } from "firebase/firestore";
+import { BsCartDash } from "react-icons/bs";
+
+import { React, useEffect, useState } from "react";
+
+function ProductsPage() {
+  const [productList, setProductList] = useState([]);
+  // MOUNT PRODUCT ON THE WEBSITE
+  useEffect(() => {
+    getProducts();
+  }, []);
+  // GET PRODUCTS FRONM DATABASE
+  const getProducts = () => {
+    const productCollection = collection(db, "ProductList");
+    getDocs(productCollection)
+      .then((res) => {
+        const products = res.docs.map((doc) => ({
+          data: doc.data(),
+          id: doc.id,
+        }));
+        setProductList(products);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
[Suggestion]
You should send the error message to users via notification/pop-up

Guide: https://dev.to/cesareferrari/how-to-display-error-messages-in-a-react-application-3g48